components/Typography/typography.html

```diff
@@ -0,0 +1,228 @@
+                        <code>
+                            <iframe
+                                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Ch1%253E%2520H1%2520heading%253C%252Fh1%253E%250A%253Ch2%253E%2520H2%2520heading%253C%252Fh2%253E%250A%253Ch3%253E%2520H3%2520heading%253C%252Fh3%253E%250A%253Ch4%253E%2520H4%2520heading%253C%252Fh4%253E%250A%253Ch5%253E%2520H5%2520heading%253C%252Fh5%253E%250A%253Ch6%253E%2520H6%2520heading%253C%252Fh6%253E"
+                                style="width: 100%; height: auto; border:0; transform: scale(1); overflow:hidden;"
+                                sandbox="allow-scripts allow-same-origin">
+                            </iframe>
+                        </code>
+                    </div>
+
+
+                    <h5 class="mt-5 mb-3 text-primary-color">Aligned Text </h5>
+
+                    <p class="mb-3 lh-10">To align the text add text-center, text-left, text-right class to wrapper
+                        element
+                    </p>
+
+                    <div style="display: block;" class="component-display flex-col">
+                        <h3 class="text-center">Text Align Center</h6>
+                            <h3 class="text-left">Text Align left</h6>
+                                <h3 class="text-right">Text Align right</h6>
+                    </div>
+
+                    <div class="code-snippet">
+                        <code>
+                            <iframe
+                                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch3%2520class%253D%2522text-center%2522%253EText%2520Align%2520Center%253C%252Fh6%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch3%2520class%253D%2522text-left%2522%253EText%2520Align%2520left%253C%252Fh6%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch3%2520class%253D%2522text-right%2522%253EText%2520Align%2520right%253C%252Fh6%253E"
+                                style="width: 100%; height: auto; border:0; transform: scale(1); overflow:hidden;"
+                                sandbox="allow-scripts allow-same-origin">
+                            </iframe>
+                        </code>
+                    </div>
+
+                    <h5 class="mt-5 mb-3 text-primary-color">Colored Text </h5>
+                    <p class="mb-3 lh-10"> For different colors you can use classes- primary-color, secondary-color,
+                        gray-color, success-color, warning-color and danger-color. </p>
```

Comment:
suggestion: give good description to understand them how to use class, avoid giving briefly ,newbie should also understand by reading

Reply:
Yes, I'll do the needed changes.